REQUIRE

```diff
@@ -2,4 +2,3 @@ julia 0.3
 Mustache
```

Comment:
Should also change to `julia 0.4-` if you're dropping 0.3 support. When you tag, best to bump the minor version of the package just in case any 0.3 bugfixes are ever necessary.